sdk/src/test/java/com/stormpath/sdk/UserProfileTest.java

```diff
@@ -0,0 +1,119 @@
+package com.stormpath.sdk;
+
+import com.stormpath.sdk.models.UserProfile;
+
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+
+import okhttp3.mockwebserver.RecordedRequest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.stub;
+import static org.mockito.Mockito.verify;
+
+public class UserProfileTest extends BaseTest {
```

Comment:
Where is the `/me` path defined? I don't see it in the framework spec. I don't think any of the other integrations have a `/me` endpoint. But, I may be missing something.